### PR TITLE
[Easylang/en] Add Easylang

### PR DESCRIPTION
**Pull request by chkas**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: easylang.html.markdown [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+#
+sum = 0
+repeat
+  s$ = input
+  until s$ = ""
+  sum += number s$
+.
+print "sum: " & sum
+#
+# input" reads a string from the "input_data" section,
```
Comment (Menelion) [inline easylang.html.markdown:161]: `"input"`

**Review by Menelion — CHANGES_REQUESTED** [review]
Thank you. Please review comments then I'll merge.

File: easylang.html.markdown (hunk 1 above)
Comment (chkas) [reply easylang.html.markdown:161]: Thank you. Well seen.

**Comment by chkas** [comment]
Thank you for reviewing. Do I need to do anything more?

**Comment by chkas** [comment]
I am not so skilled with Github. Was the close now a mistake.

File: easylang.html.markdown [hunk 2]
```diff
@@ -0,0 +1,220 @@
+---
+language: Easylang
+contributors:
+    - ["chkas", "https://github.com/chkas"]
+---
```
[the remark this answers is not in the document]
Comment (chkas) [reply easylang.html.markdown:5]: filename: easylang.el

**Comment by Menelion** [comment]
@chkas Yes, closing was a mistake. Could you reopen please? If not, I will care of it myself and merge the branch.

**Comment by chkas** [comment]
It cannot be reopened because the repository has been deleted. I can make a new pull request if that is ok.

**Comment by chkas** [comment]
I just submitted a new pull request that I will not close.

**Pull request closed without merging**